src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -3021,10 +3037,12 @@ public function get_attendee( $attendee, $post_id = 0 ) {
 	 */
 	protected function get_cancelled( $ticket_id ) {
 		$denied_orders = Tribe__Tickets__Commerce__PayPal__Order::find_by( array(
-			'ticket_id'   => $ticket_id,
-			'post_status' => Tribe__Tickets__Commerce__PayPal__Stati::$denied,
+			'ticket_id'      => $ticket_id,
+			'post_status'    => Tribe__Tickets__Commerce__PayPal__Stati::$denied,
 			'posts_per_page' => -1,
-		) );
+		), [
```

Comment:
* :no_entry_sign: Multi-line function call not indented correctly; expected 12 spaces but found 8
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 9d58f5c71a287606732cf5a910dde8ba</sub>